person.rb

```diff
@@ -4,17 +4,17 @@
 
 class Person < Nameable
```

Comment:
- [ Required ] It would be better if you add a method `add_rental` that can be used any time a person is renting a book, this method will receive as arguments a `book` and a `date` object. Currently, there is no method in your class that can be used to achieve this. You can achieve this by using the intermediate `Rental` class as shown in the m method below.

```ruby
  def add_rental(book, date)
     Rental.new(date, person, self)
  end
```
## Project requirements
![Screenshot from 2023-02-11 00-28-03](https://user-images.githubusercontent.com/16392046/218203155-45307821-7d6b-43bd-bbdc-4b884739d146.png)

Reply:
I was a little confused in this part. I used Rental.new(date, book, self) instead 😅